bin/node/runtime/src/lib.rs

```diff
@@ -627,6 +627,7 @@ frame_election_provider_support::generate_solution_type!(
 parameter_types! {
 	pub MaxNominations: u32 = <NposSolution16 as frame_election_provider_support::NposSolution>::LIMIT as u32;
 	pub MaxElectingVoters: u32 = 10_000;
+	pub MaxWinners: u32 = 1000;
```

Comment:
Likes give this a name with a bit more context here -- `MaxActiveValidators`, `MaxStakingWinners` or such? It won't look as nice though.